src/CMakeLists.txt

```diff
@@ -87,6 +89,21 @@ set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} ${c_cxx_flags}")
 if(BUILD_FAST_TOKENIZERS)
   # FastTokenizers use static MSVC runtime, hence we have to use it as well
   set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
+
+  if(WIN32 AND CMAKE_BUILD_TYPE MATCHES Debug)
+    set(CMAKE_DEBUG_POSTFIX "d")
+    foreach(
+        flag_var
+        CMAKE_CXX_FLAGS_DEBUG
+        CMAKE_C_FLAGS_DEBUG)
+        if(${flag_var} MATCHES "/MD")
+          string(REGEX REPLACE "/MD" "/MTd" ${flag_var} "${${flag_var}}")
+        elseif(NOT (${flag_var} MATCHES "/MTd"))
+          set(${flag_var} "${${flag_var}} /MTd")
```

Comment:
I believe we need to avoid building with /MT as we finally building tokenizers as DLL, which should have been linked against dynamic MSVC runtime